### PR TITLE
Complete UI MVP

### PR DESCRIPTION
**Pull request by LachelleF**
Corrected UI issues. Developed to MVP. Will go back after foundational, core deliverables are complete and refine.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -4,12 +4,12 @@ import PriceAndPromoContainer from "./PriceAndPromoContainer.jsx";
 import PriceImage from "./PriceImage.jsx"
 import axios from 'axios';
 
-const AppWrapper = styled.div`
-  width: 370.5px;
-  height: 431.54px;
-`;
 const AppStyled = styled.div`
-  background-color: #1b1e1b;
+  font-family: 'Lucida Grande', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (Chris-Hesterman) [inline client/components/App.jsx:8]: Have you found this to be a good substitue for Sofia Pro?  that would be great!

File: client/components/App.jsx [hunk 2]
```diff
@@ -63,16 +63,14 @@ class App extends React.Component {
       return (null);
```
Comment (Chris-Hesterman) [inline client/components/App.jsx:63]: An interesting alternative to having an initial state.  How does it look in action?

File: client/components/Platforms.jsx [hunk 3]
```diff
@@ -2,16 +2,15 @@ import React from 'react';
...
+height: 25px;
 `;
 
 const IconStyled = styled.div`
-  color: white;
-  width: 18px;
-  height: 17.95px;
-  margin: 0;
-  padding: 0;
+color: white;
```
Comment (Chris-Hesterman) [inline client/components/Platforms.jsx:12]: No point in setting a color, it is a .png image.  if you need a different color I can whip one up for you.

File: client/components/App.jsx (hunk 1 above)
Comment (LachelleF) [reply client/components/App.jsx:8]: I have, haven't had any issues so far. I think it looks pretty good.

File: client/components/App.jsx (hunk 2 above)
Comment (LachelleF) [reply client/components/App.jsx:63]: I really like it. Everything I've done up til now I implemented very differently. It's been a bit of a learning curve. Was encouraged to try it by Matt and a SEIR.

File: client/components/Platforms.jsx (hunk 3 above)
Comment (LachelleF) [reply client/components/Platforms.jsx:12]: oh yes! super great point, will edit that accordingly :)

**Pull request merged**